package.json

```diff
@@ -81,6 +81,8 @@
     "worker-loader": "^3.0.8"
   },
   "engines": {
-    "node": ">=14.17.0"
+    "node": ">=14.17.0",
+    "npm": "please-use-yarn",
```

Comment:
Fun trick, I like it! What are the limitations on the message? I'd prefer something more humane like "Please use yarn instead of npm" if possible

Reply:
I don't think that there are any limitations in a sense that we can put any string we want, I will update.